graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -124,6 +127,7 @@ class HeterogeneousLoaderConfig:
     num_output_files: int
     output_prefix: str
     precomputed_transformations: dict
+    is_homogeneous: bool
```

Comment:
Is there any chance this get instantiated without a value for `is_homogeneous`?

Reply:
I added a default `False` value for it. We expect most of the graph input will not be homogeneous graph.